main/man-pages/template.py

```diff
@@ -54,6 +54,11 @@ def install(self):
         self.rm("sgetspent_r.3")
         self.rm("ulckpwdf.3")
 
+    # Useless, pull in bash and other stuff we don't want
```

Comment:
self.uninstall("usr/bin")

Reply:
hm actually nevermind that, since we need to uninstall the individual manpages too, this is fine